### PR TITLE
Issue# 13

### LINKED ISSUE #13: Update pom.xml to point to Unleash repo instead of personal repo
### Describe the feature request

Update pom.xml to point to Unleash repo instead of personal repo
Currently scm/connection, scm/developerConnection and scm/url all points to a personal repo.
This needs to be updated to point to Unleash repo

### Background

_No response_

### Solution suggestions

Update pom.xml

### PR DESCRIPTION
**Pull request by praveenpg**
https://github.com/Unleash/unleash-spring-boot-starter/issues/13 Update pom.xml to point to Unleash repo

Update pom.xml to point to Unleash repo instead of personal repo
Currently scm/connection, scm/developerConnection and scm/url all points to a personal repo.
This needs to be updated to point to Unleash repo

## About the changes

Closes #13 

### Important files

/pom.xml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <spring.boot.version>2.7.4</spring.boot.version>
+        <spring.boot.version>2.7.5</spring.boot.version>
```
Comment (thomasheartman) [inline pom.xml:36]: Does this impact anything/anyone in a breaking fashion? Or does the package require a minor bump for this?

**Review by thomasheartman — APPROVED** [review]
Ah, good catch! This looks great to me 🚂 The only comment I have is that it also updates the `spring.boot.version`, and I don't know how that would affect consumers. Is that something that could case incompatibilities or something?

File: pom.xml (hunk 1 above)
Comment (praveenpg) [reply pom.xml:36]: @thomasheartman  This is a bugfix release from spring which is backward compatible with 2.7.4
https://spring.io/blog/2022/10/20/spring-boot-2-7-5-available-now. 
I think a version bump is a good idea.

Comment (praveenpg) [reply pom.xml:36]: @thomasheartman I will revert back the version change and create a separate PR with that

**Comment by praveenpg** [comment]
@thomasheartman How do I make a release of the first version? I have done releases from my personal repo, not sure how I do it from this repo.

**Comment by thomasheartman** [comment]
That's ... a very good question. How did you do it from your personal repo? Regardless, I feel like @chriswk is probably the one with the most knowledge about java release processes around.

by the way, is there a reason you haven't merged this yet? You've got the required access rights, right?

**Comment by thomasheartman** [comment]
@praveenpg , did you get anywhere on the releases? We got a user [asking about releases in a discussion](https://github.com/orgs/Unleash/discussions/2533) the other day, so the work you're doing definitely serves some needs 😄

**Comment by praveenpg** [comment]
@thomasheartman As I had mentioned earlier, I am not sure how do I make a release from the Unleash repo. I have done from my personal repo.

**Pull request merged**